Source/Core/Common/Hash.cpp

```diff
@@ -530,4 +532,19 @@ void SetHash64Function()
     ptrHashFunction = &GetMurmurHash3;
   }
 }
+
+std::string MakeGameID(std::string file_name)
```

Comment:
```suggestion
std::string MakeGameID(std::string_view file_name)
```

Reply:
You forgot to change this. (don't forget to include `<string_view>` if necessary)